### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ViscousGuy**
Fix minor redundancy and improve clarity.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -116,12 +116,12 @@ To install Devika, follow these steps:
 3. Create a virtual environment and install the required dependencies:
    ```bash
    uv venv
+   source .venv/bin/activate
```
Comment (nalaso) [inline README.md:119]: It doesn't work for windows users.

Comment (ViscousGuy) [reply README.md]: Fixed it.

**Comment by ARajgor** [comment]
resolve conflicts

**Comment by NoGuyFast** [comment]
this changed from yesterday:
7. Now, for frontend, open a new terminal and navigate to the ui directory
(...)
bun run start
8. Access the Devika web interface by opening a browser and navigating to http://127.0.0.1:3001

I did a git pull today and saw the changes in the readme
_bun run start_ doesn't work at all for me, I still have to use _bun run dev_
its still http://localhost:3000
127.0.0.1 doesn't work at all because port 3000 is mapped to the IPV6 localhost - system: windown 10

It is possible to map the port 3000 to IPV4?

**Comment by ARajgor** [comment]
> this changed from yesterday:
> 7. Now, for frontend, open a new terminal and navigate to the ui directory
> (...)
> bun run start
> 8. Access the Devika web interface by opening a browser and navigating to http://127.0.0.1:3001
> 
> I did a git pull today and saw the changes in the readme
> _bun run start_ doesn't work at all for me, I still have to use _bun run dev_
> its still http://localhost:3000
> 127.0.0.1 doesn't work at all because port 3000 is mapped to the IPV6 localhost - system: windown 10
> 
> It is possible to map the port 3000 to IPV4?

Can you share a `bun run start` log. And it's running on IPv6 ?? Share the screenshot.

**Pull request closed without merging**